### PR TITLE
[release] move release testing end to end script to main ray repo

### PR DESCRIPTION
**Pull request by krfricke**
## Why are these changes needed?

cc @ericl @simon-mo @rkooo567

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by krfricke** [comment]
As can be seen here (branch **release-testing**) https://buildkite.com/ray-project/periodic-ci-ray-repo this works.

After merging we can disable the old releaser repo.

**Comment by krfricke** [comment]
FYI, contents of the files are the same as in the releaser repo, with just minor alterations (linting and one change to the default RAY_PROJECT)

File: release/.buildkite/build_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
+import copy
```
Comment (ericl) [inline release/.buildkite/build_pipeline.py:1]: Is it possible to put this under `ci/` instead of making a new toplevel dir?

Comment (simon-mo) [reply release/.buildkite/build_pipeline.py:1]: release is already a top level dir with all the release testing config?

File: release/config_generator.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
+<!doctype html>
```
Comment (simon-mo) [inline release/config_generator.html:1]: should we upgrade this to buildkite input step?

**Review by ericl — APPROVED** [review]

File: release/config_generator.html (hunk 2 above)
Comment (krfricke) [reply release/config_generator.html:1]: Let's do this in a separate PR

**Pull request merged**